packages/auto-label/README.md

```diff
@@ -0,0 +1,21 @@
+auto-label
+
```

Comment:
we should get your PR landed that makes the README slightly prettier :laughing:, I think it just has a linting issue:

https://github.com/googleapis/repo-automation-bots/pull/230